pytest_splunk_addon/standard_lib/cim_tests/test_templates.py

```diff
@@ -16,6 +17,63 @@ class CIMTestTemplates(object):
         - Not Allowed Fields should not be extracted 
         - TODO 
     """
+
     logger = logging.getLogger("pytest-splunk-addon-cim-tests")
 
-    # TODO: Add Test scenarios 
+    def test_eventtype_mapped_datamodel(
+        self, splunk_search_util, record_property, caplog
+    ):
+        """
+        This test case check that event type is not be mapped with more than one data model 
+
+        Args:
+            splunk_search_util (SearchUtil): Object that helps to search on Splunk.
+            record_property (fixture): Document facts of test cases.
+            caplog (fixture): fixture to capture logs.
+        """
+        search = """
+            | tstats count from datamodel=Alerts  by eventtype | eval dm_type="Alerts" 
+            | append [| tstats count from datamodel=Application_State  by eventtype | eval dm_type="Application_State"] 
```

Comment:
Can we create this search query dynamically using a list? It would provide an easy to future developers who want to change the search query.

Reply:
Done.